pkg/reconciler/openshift/rbac/rbac.go

```diff
@@ -85,6 +92,21 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ns *corev1.Namespace) pk
 	return nil
 }
 
+func (r *Reconciler) ensureTrustedCABundle(ctx context.Context, ns *corev1.Namespace) error {
+	logger := logging.FromContext(ctx)
+	logger.Info("finding configmap: %s/%s", ns.Name, trustedCABundle)
+	cfgInterface := r.kubeClientSet.CoreV1().ConfigMaps(ns.Name)
+	_, err := cfgInterface.Get(ctx, trustedCABundle, metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+	if err != nil && errors.IsNotFound(err) {
+		logger.Info("creating configmap", trustedCABundle, "ns", ns.Name)
+		return createTrustedCABundleConfigMap(ctx, cfgInterface, trustedCABundle, ns.Name)
+	}
+	return nil
+}
+
```

Comment:
shall we move this into the post reconcile of OpenShift Extension for TektonPipeline Reconciler.
if we add this functionality here, it might not be intutive for other users to look for this functionality in rbac controller.

Reply:
I'd defer to @vdemeester for this since this comes from #190

Reply:
@nikhil-thomas the problem with doing this in an extension is not having access to a `(r *Reconciler)` instance which comes in handy when trying to inject or create objects in the cluster. I think this was the reason @vdemeester put this here (maybe)